assets/js/components/data/cache.js

```diff
@@ -27,7 +27,7 @@ import { cloneDeep } from 'lodash';
 import { getStorage } from '../../util/storage';
 import { stringifyObject } from '../../util/stringify';
 
-export const STORAGE_KEY_PREFIX = 'googlesitekit_legacy_';
+export const STORAGE_KEY_PREFIX = `googlesitekit_legacy_${ global.GOOGLESITEKIT_VERSION || '' }`;
```

Comment:
Unfortunately, looks like DefinePlugin can't replace things in string templates. So this doesn't work and we need to update it to the following:

```suggestion
export const STORAGE_KEY_PREFIX = 'googlesitekit_' + ( global.GOOGLESITEKIT_VERSION || '' );
```